deepcell_toolbox/processing_test.py

```diff
@@ -95,12 +95,17 @@ def test_histogram_normalization():
 
         # test negative coordinates don't get clipped
         negative_coords = (img < 0).nonzero()
-        if len(preprocessed[negative_coords]):
+        if len(preprocessed[negative_coords]) > 0:
             assert (preprocessed[negative_coords] >= 0).all()
 
         # test legacy version
         legacy_img = processing.phase_preprocess(img)
         np.testing.assert_equal(legacy_img, preprocessed)
+    
+    # test constant value array raises error
+    with pytest.raises(ValueError):
+        image = np.ones((1, 32, 32, 1))
+        processing.histogram_normalization(image)
```

Comment:
If you look at line 129 below this, you can see a test I added for blank images for the thresholding step. I think as written, this will currently fail on two channel images where one channel is completely blank. 

However, the function runs as expected for blank images, it's only non-zero constant images that cause the bug. Maybe changing the value error that gets raised?

Or instead of raising a value error, just returning np.ones() of the same shape? Not sure what the best option is for totally blank.

Reply:
OK so we don't need to worry about blank images, but non-zero constant value images are the problem (agreed, this is unlikely). I'll check if the array is non-zero and if so, throw a value error.